local_setup.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
This is unlikely to cause huge issues either but if you do `#!/usr/bin/env bash` [it makes it more portable.](https://en.wikipedia.org/w/index.php?title=Shebang_(Unix)&oldid=878552871#Portability)